apps/pr-deploy-site/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <ul>
+      <li><a href="./fabric-website-resources/dist/index.html">Fabric Demo App</a></li>
```

Comment:
Do you think there's benefit in defaulting to `target="_blank"` for these or should I just commit to holding `CTRL/CMD`?

Reply:
Maaaaybe? I like the back button? :)

Reply:
I think better is if our apps share a single header that can flip between these apps